MangoPay/PayInPaymentDetailsCard.php

```diff
@@ -8,8 +8,9 @@
 class PayInPaymentDetailsCard extends Libraries\Dto implements PayInPaymentDetails
 {
     /**
-     * CardType { CB_VISA_MASTERCARD, AMEX }
+     * CardType
      * @var string
+     * @see \MangoPay\CardType
```

Comment:
```suggestion
     * @see \MangoPay\CardType
     * @phpstan-var \MangoPay\CardType::*
```
This will allow tools like phpstan to warn the dev when not using the enum